corehq/apps/case_search/utils.py

```diff
@@ -103,3 +104,20 @@ def _add_case_property_queries(self):
                 to_remove = re.escape(removal_regex.regex)
                 value = re.sub(to_remove, '', value)
             self.search_es = self.search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
+
+    def _add_daterange_queries(self):
```

Comment:
nit, can you put this above `_add_case_property_queries` to keep things in the order they are called?

Reply:
Sure